test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -0,0 +1,51 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /group/:groupId/join', () => {
+  let user;
+
+  beforeEach(() => {
+    return generateUser().then((generatedUser) => {
+      user = generatedUser;
+    });
+  });
+
+  context('Guilds', () => {
+    it('inviting to guild doesn\'t give basilist to inviter', async () => {
+      let groupName = 'Test Guild';

```

Comment:
Same answer as before, when I add the new tests the plan is to use more beforeEach blocks for common logic.